Formula/zsh-syntax-highlighting.rb

```diff
@@ -1,7 +1,7 @@
 class ZshSyntaxHighlighting < Formula
   desc "Fish shell like syntax highlighting for zsh"
   homepage "https://github.com/zsh-users/zsh-syntax-highlighting"
-  url "https://github.com/zsh-users/zsh-syntax-highlighting/archive/0.4.1.tar.gz"
+  url "https://github.com/zsh-users/zsh-syntax-highlighting.git", tag: "0.4.1"
```

Comment:
:+1: Will update later when I get home.

Why does it need a tag _and_ a revision, btw?